### ISSUE TITLE
[Bug]: Doesn't work on Dashie (android app) (because of iframe)

### ISSUE THREAD

**Issue by tbrasser** [Lint-Free-Technology/uix#558]
### My Home Assistant Version

Core: 2026.10.0.dev202608280452
Frontend: 20260828.0.dev
UIX: v8.2.0-beta.1 (now patched by llm to make it work)

### My UIX installation method

HACS

### My lovelace configuration method

YAML

### What I am doing

Trying to use uix on android wall tablet using Dashie

### What I expect to happen

uix to apply styling

### What happened instead

uix doesn't apply styling (when loaded in iframe as Dashie apparently does)

### Minimal steps to reproduce

Use recent uix in iframe (e.g. Dashie for home assistant)

### Include any yaml code here

```YAML

```

### Error messages from the browser console

```console

```

### By checking each box below I indicate that I ...

- [x] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).
- [x] Have made sure I am using the latest version of the integration.
- [x] Have searched Home Assistant logs for any relevant UIX errors or warnings.
- [x] Understand that failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by dcapslock** [comment]
Hi, this report is lacking information and confusing as it mentions even that it now works. Can you please clarify the issue, provide setup information, link to the any integration or plugin you are using. Thank you.

**Comment by dcapslock** [comment]
Hi, is you report specific to 8.2.0-beta.1? If so can you confirm was working with 8.1.0? Once you do I can confirm a fix as the code for custom panels would have broken a standard Dashboard being hosted in an iFrame.

**Comment by dcapslock** [comment]
Let me know the issue fixed matched your issue. If not please provide more information to further triage.

**Comment by tbrasser** [comment]
Sorry for not responding sooner.

My HA/dashboard has been broken for a while unrelated to UIX.
So I didn't know exactly when the 'breaking' change was introduced. Since I only noticed after I got everything back up (and updated)

Let me try the newest release and report back.

**Comment by tbrasser** [comment]
It seems to work! Great!

**Comment by dcapslock** [comment]
Great. Thank for the update.
